### PR TITLE
fix(CRM): Missing permissions for Sales User

### PR DESCRIPTION
**Pull request by haiderk0987**
> Please provide enough information so that others can review your pull request:

> Explain the **details** for making this change. What existing problem does the pull request solve?

> Screenshots/GIFs

### CODE REVIEW

**Comment by SaiFi0102** [comment]
Wrong branch Mr

File: erpnext/crm/doctype/sales_stage/sales_stage.json [hunk 1]
```diff
@@ -33,10 +36,20 @@
...
    "share": 1,
    "write": 1
+  },
+  {
+   "email": 1,
+   "export": 1,
+   "print": 1,
+   "report": 1,
+   "role": "Sales User",
+   "select": 1,
```
Comment (SaiFi0102) [inline erpnext/crm/doctype/sales_stage/sales_stage.json:47]: Please add "Read" permission for Sales Stage as well
Add "Read" and "Select" permission for "Sales User (Read Only)" as well

File: erpnext/hr/doctype/driver/driver.json [hunk 2]
```diff
@@ -159,13 +162,23 @@
    "role": "HR Manager",
    "share": 1,
    "write": 1
+  },
+  {
+   "email": 1,
+   "export": 1,
+   "print": 1,
+   "report": 1,
+   "role": "Sales User",
```
Comment (SaiFi0102) [inline erpnext/hr/doctype/driver/driver.json:171]: Add "Select" permission for "Sales User (Read Only)" as well

**Review by SaiFi0102 — CHANGES_REQUESTED** [review]

File: erpnext/crm/doctype/sales_stage/sales_stage.json (hunk 1 above)
Comment (haiderk0987) [reply erpnext/crm/doctype/sales_stage/sales_stage.json:47]: done

File: erpnext/hr/doctype/driver/driver.json (hunk 2 above)
Comment (haiderk0987) [reply erpnext/hr/doctype/driver/driver.json:171]: done

**Pull request merged**